examples/vl3_basic/scripts/demo_vl3_ipam.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
```

Comment:
Don't make a whole new script for the integration with the ipam server.  You'll need to test with the cnns-system installation.  To test with your version of the nse you'd modify the cnns-system/dependencies.env to set
NSE_HUB=<your image ORG> 
NSE_TAG=<your image tag>